contract/test/test-build-proposal.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/order -- https://github.com/endojs/endo/issues/1235 */
+import { test } from './prepare-test-env-ava.js';
+import { execa } from 'execa';
+import { promises as fs } from 'fs';
+import { makeCompressFile } from './utils.js';
+
+test.before(t => (t.context.compressFile = makeCompressFile(fs.readFile)));
+
+test('proposal builder generates compressed bundles less than 1MB', async t => {
+  const { stdout } = await execa('agoric', [
```

Comment:
This doesn't seem to make use of any `execa` feature that isn't provided by node's `child_process`. Am I missing something?

Reply:
`execa` dep removed